test/apns/token_bearer_test.exs

```diff
@@ -1,12 +1,18 @@
 defmodule Sparrow.APNS.TokenBearerTest do
   use ExUnit.Case
+  alias Helpers.SetupHelper, as: Tools
+
+  import Mox
+  setup :set_mox_global
+  setup :verify_on_exit!
 
   @key_id "KEYID"
   @team_id "TEAMID"
   @p8_file_path "token.p8"
   @refresh_time 100
   @token_id :token_id
   setup do
+    Tools.passthrough_h2
```

Comment:
Same here, we can make this a separate, individual setup block:
```
setup Tools.passthrough_h2
```

Reply:
same as above